templates/all-hands-issue-template.md

```diff
@@ -2,8 +2,11 @@
 
 Please add your agenda items before the call.
 
-Endeavour      | Moderator            | Notetaker | Time (PST - UTC - CET) | Pad
+Endeavour      | Moderator            | Notetaker | Time (PST - UTC - CEST) | Pad
 :------------: | :-------------: | :-------: | :--------------------: | :----:
-All Hands Call | $MODERATOR    | $NOTETAKER  | 9:00 **16:00** 17:00  | [agenda and notes]($CRYPTPAD)
+All Hands Call | $MODERATOR    | $NOTETAKER  | 9:00 **16:00** 18:00  | [agenda and notes]($CRYPTPAD)
+
+Every last Monday of the month, we will have three 5-minute lightning talks. The mini presentations will serve as another way to showcase some of the cool stuff the IPFS community is working on! 😎
```

Comment:
Hi Portia - I think you read my PR the opposite way. The plan is actually to have an open "show and tell" where we ask for signups and agenda at the beginning of the meeting and any attendee can do a quick demo, explanation, call for contribution, etc. The thought is this will be a more engaging and participatory forum compared to the other 3 all hands per month that are more presentation based.

Reply:
Ah, ok. So for "Show and Tell" participants will sign up during the beginning of the call and not before.